Fix coreBoundarySurface preset advanced parameters labels

klimaj opened the pull request:
This PR:
-  fixes minor TODOs in `presets.py`
-  fixes `coreBoundarySurface` preset advanced parameter labels
![Screen Shot 2022-08-31 at 11 20 19 PM](https://user-images.githubusercontent.com/7121657/187849535-ef3d1fb8-091c-4cef-a619-4b1c696e8500.png)

klimaj committed:
Fix advanced labels

klimaj committed:
Get full-atom scorefxn

klimaj committed:
Fix TODO

klimaj committed:
Merge branch 'main' of https://github.com/RosettaCommons/pyrosetta_viewer3d into klimaj/presets

klimaj committed:
git pull origin main

ajasja reviewed (approved):
Nice! LGTM. 
Also I had the chance to try the interactive controls from inside the US. The latency is sooooo much better :)

ajasja committed:
Fix matplotlib dependency

On gooogle colab, which is running python 3.7.13 by default, matplotlib 3.5 works best as dependency. 3.5.2 raises an erorr.

klimaj commented:
@ajasja Great to hear! Thanks for the review!

klimaj commented:
@ajasja Great to hear! Thanks for the review!
